src/definitions/modules/toast.less

```diff
@@ -119,7 +158,34 @@
     font-size: @toastIconFontSize;
     opacity: @toastIconOpacity;
 }
+
+.icon.flat.toast > .icon:not(.close) {
```

Comment:
Where does this `flat` class come from ?

Reply:
I invented it. This will override the normal 0.9 opacity toast to full opacity toast component.
Personally i like that better, but i did not want to completely break the original authors idea which only sets opactity to 1 when you hover over the toast.
Anyway, the message class will always have opacity 1, so there was no need to invent a flat class aswell there.

Reply:
`className: {toast:'ui flat toast'}`
![image](https://user-images.githubusercontent.com/18379884/46217315-ee1d7a80-c341-11e8-90b9-c6f4a7d4c62c.png)

`className: {toast:'ui toast'}` (default)
![image](https://user-images.githubusercontent.com/18379884/46217233-bca4af00-c341-11e8-93a7-fe6f33dfd534.png)

To be honest: Even with the increased 0.9 opacity, this looks unuseful to me. I suggest having the toast always have opacity 1 as default like message and we can remove the flat class there

Reply:
This is opacity: 0.97. If we really want to keep lower opacity, it should be raised to >=0.95 again imho. The idea of a toast component is to be displayed over existing content. In most cases, i guess 0.9 is still to low to read the toast-content nicely.
![image](https://user-images.githubusercontent.com/18379884/46217655-dc88a280-c342-11e8-9cc1-5c6d84db088f.png)

Reply:
I do agree with this.

Reply:
In this case why don't we let the user to set an opacity as a toast parameter ?
```
opacity: 0.8 //set as 1 by default
```

Reply:
ok, i'll remove the 'flat' class and will add an option for opacity (default 1)
Please keep in mind, whenever somebody is using something lower than 1 the fadeIn options of SUI will always animate to Opacity 1 and then immediatly the  toast would change to the lower opacity (otherwise the whole css-fade animation would need a change aswell, which i think is overkill) Here is ax example how it looked with the original fix 0.8 opacity
![toast_org_fadein](https://user-images.githubusercontent.com/18379884/46276442-63669680-c560-11e8-9c2b-dae01edc92cc.gif)

Reply:
Yeah I saw that too. But letting the user the ability to change the opactity is nice imho.